pkg/storage/s3/s3.go

```diff
@@ -40,10 +40,6 @@ func New(s3Conf *config.S3Config, timeout time.Duration, options ...func(*aws.Co
 		return nil, errors.E(op, err)
 	}
 
-	// Remove anonymous credentials from the default config so that
-	// session.NewSession can auto-resolve credentials from role, profile, env etc.
-	awsConfig.Credentials = nil
```

Comment:
Ah, this is interesting. I wonder if roles, profiles, and ENV settings will break now?

Reply:
Okay, I think I understand the problem. Line 38 should be using [this](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/aws/external#LoadDefaultAWSConfig)

Reply:
Okay, nevermind - [this](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials) clearly says that roles, profiles, env etc should be covered by `LoadDefaultConfig`